config.yaml

```diff
@@ -0,0 +1,16 @@
+baseURL: https://cpssd.net
+disablePathToLower: true
```

Comment:
y tho

Reply:
This was some time ago that I did it. I believe I imported the site from Jekyll and it was the default setting. I believe this basically means that if a file is called `SomeBlogPost.md`, this would generate `SomeBlogPost.html` from it instead of `someblogpost.html`. I can remove that line if you want.

Reply:
It sounds like nbd